lib/utils.js

```diff
@@ -141,12 +142,12 @@ let util = module.exports = {
 									ExpressionAttributeValues: {
 										":endTime": Date.now(),
 										":token": payload.__cron.ts,
-										":status": status,
+										":status": undefined,
 										":log": zlib.gzipSync(JSON.stringify(err.message)),
 									},
 									"ReturnConsumedCapacity": 'TOTAL'
-								}
-								if (typeof options.releaseLockOnError == "funtion") {
+								};
+								if (typeof options.releaseLockOnError === "function") {
```

Comment:
This command should be changed so we have just `let command;`
Then below we have an if it's a function, assign that, otherwise do the DDB thing. The reason for that being that we use gzipSync, so if it happens to be a function, we should save on processing by not building the DDB stuff and using gzip.

Reply:
the `let command =` for this is up above.  this line is just a hook to the calling lib to transform the command just before it is run

Reply:
What I mean:

`let command;

if (typeof options.releaseLockOnError === 'function') {
    command = … stuff …
} else {
    command = {
        TableName… Key… etc…`
    };
}
await dyanmodb.docClient.update(command).promise();